src/Markup/Avalonia.Markup.Xaml/PortableXaml/AvaloniaXamlObjectWriter.cs

```diff
@@ -27,16 +27,36 @@ public class AvaloniaXamlObjectWriter : XamlObjectWriter
                                                 nameScope);
         }
 
+        public static AvaloniaXamlObjectWriter Create(
+            XamlSchemaContext schemaContext,
+            AvaloniaXamlContext context,
+            IAmbientProvider parentAmbientProvider)
```

Comment:
Wouldn't it make more sense to just modify the `Create` method above to take a `IAmbientProvider parentAmbientProvider = null` parameter rather than duplicating the code?

Reply:
I wasn't sure how strict this project is pertaining to binary compatibility. Adding parameter would break it, so I added new method. It can definitely be done as you said :)